src/coreclr/jit/codegenxarch.cpp

```diff
@@ -2260,6 +2300,12 @@ void CodeGen::genCodeForTreeNode(GenTree* treeNode)
             // Do nothing; these nodes are simply markers for debug info.
             break;
 
+#if defined(TARGET_AMD64)
+        case GT_CCMP:
+            genCodeForCCMP(treeNode->AsCCMP());
+            break;
+#endif
```

Comment:
What prevents this from being supported on x86?

Reply:
IIUC, APX is only available in Intel 64-bit mode.